File: Dsa-day67.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        int[] freq = new int[101];
        int maxFreq = 0, total = 0;
        for (int n : nums) {
            freq[n]++;
            if (freq[n] > maxFreq) maxFreq = freq[n];
        }
        for (int n : nums) {
            if (freq[n] == maxFreq) total++;
        }
        return total;
    }
}

File: Dsa-day67.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        freq = {}
        max_freq = 0
        total = 0
        for n in nums:
            freq[n] = freq.get(n, 0) + 1
            if freq[n] > max_freq:
                max_freq = freq[n]
        for n in nums:
            if freq[n] == max_freq:
                total += 1
        return total
